orc8r/cloud/helm/orc8r/charts/secrets/templates/_helpers.tpl

```diff
@@ -17,3 +17,28 @@ app.kubernetes.io/managed-by: helm
 app.kubernetes.io/part-of: magma
 helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
 {{- end -}}
+
+{{- define "orchestrator-config-template" -}}
+useGRPCExporter: true
+prometheusGRPCPushAddress: "{{ .Values.fullReleaseName }}-prometheus-cache:9092"
```

Comment:
Ah ok. I was basing this off of how we deploy on minikube in which the secrets chart is deployed with release name `orc8r-secrets`. Will change though since that's not normal.